### PR TITLE
[BugFix] fix several bugs of pipe

### PR DESCRIPTION
**Pull request by murphyatwork**
Why I'm doing:

What I'm doing:
1. Check insert privilege when creating pipe
2. Fix display `pipes.created_time` 
3. Fix display `pipes.last_error` 

Fixes #issue

## What type of PR is this:

- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/common/util/DateUtils.java [hunk 1]
```diff
@@ -172,6 +173,10 @@ public static String convertDateTimeFormaterToSecondFormater(String datetime) {
         return convertedDatetime;
     }
 
+    public static String formatTimeStampInSeconds(long timestampInSeconds) {
```
Comment (imay) [inline fe/fe-core/src/main/java/com/starrocks/common/util/DateUtils.java:176]: The timestamp is one word, so it should be `formatTimestampInSeconds`

File: fe/fe-core/src/main/java/com/starrocks/common/util/DateUtils.java [hunk 2]
```diff
@@ -73,6 +73,7 @@ public class DateUtils {
             .addSerializationExclusionStrategy(new GsonUtils.HiddenAnnotationExclusionStrategy())
             .addDeserializationExclusionStrategy(new GsonUtils.HiddenAnnotationExclusionStrategy())
             .enableComplexMapKeySerialization()
+            .disableHtmlEscaping()
```
[the remark this answers is not in the document]
Comment (mofeiatwork) [reply fe/fe-core/src/main/java/com/starrocks/common/util/DateUtils.java:76]: It's only used for pipe, no others use it.

**Comment by mofeiatwork** [comment]
@Mergifyio rebase

**Review by Seaven — APPROVED** [review]

**Review by satanson — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/load/pipe/Pipe.java [hunk 3]
```diff
@@ -496,16 +498,26 @@ public void suspend() {
...
 
                 if (!runningTasks.isEmpty()) {
-                    runningTasks.clear();
                     LOG.info("suspend pipe {} and clear running tasks {}", this, runningTasks);
+                    runningTasks.clear();
                 }
                 loadStatus.loadingFiles = 0;
+
+                // Change LOADING files to UNLOADED
+                if (CollectionUtils.isNotEmpty(loadingFiles)) {
```
Comment (packy92) [inline fe/fe-core/src/main/java/com/starrocks/load/pipe/Pipe.java:517]: What does this mean?

**Pull request merged**